configs/antv.json

```
@@ -2,73 +2,85 @@
   "index_name": "antv",
   "start_urls": [
     {
-      "url": "https://antvis.github.io/zh",
+      "url": "https://antv.vision/zh.*",
```

```suggestion
      "url": "https://antv.vision/zh",
```
Same review, Let's use URL instead of regex. This start_url will have the same having as using the regex